src/ert/analysis/_es_update.py

```diff
@@ -36,6 +36,7 @@
 from ..config.analysis_module import ESSettings, IESSettings
 from . import misfit_preprocessor
 from .event import (
+    AnalysisCSVEvent,
```

Comment:
Perhaps we internally should use a more informative name then CVS for this set of data?  Imo its only really CSV when saved to disk

Reply:
Good point, how about `AnalysisTabularEvent`? Or maybe `AnalysisDataEvent`?